syft/math.py

```diff
@@ -45,3 +45,41 @@ def matmul(tensor1, tensor2):
         return dot(tensor1, tensor2)
     else:
         return TensorBase(np.matmul(tensor1.data, tensor2.data))
+
+
+def ceil(tensor):
+    """
+    Returns the ceilling input tensor,element wise .
+
+    Ceilling of an input scalar is the smallest integer such as :
+    for each floating pount number x : a >= x
+
+    Behavior is independent of a tensor's shape.  
+
+    :input: TensorBase tensor\n
+    :return: TensorBase tensor    
```

Comment:
Sorry I'm somehow nitpicking :D
Put a space after `,` (lines 52, 69, 85 in this file, lines 23, 25, 30, 31 in `test_math.py`) and remove space before period and colon (lines 52, 54, 64 in this file).
Also try to make the code `pep8`-compliant.